### DISCUSSION TITLE
Can this be accomplilshed with just registry entries?

### DISCUSSION THREAD

**Discussion by DJGray** [Open-Shell/Open-Shell-Menu#1623] [Q&A] [answered]
I love Classic/Open Shell and have used it for many years at home (and a few at work). I was recently scolded for putting Open Shell on my work computer because I deeply dislike the WinX start menu. Open Shell was removed and I have been forbidden to reinstall it. 

I'm wondering if I can accomplish the same effect using only registry entries.  Can this be done? I've searched extensively for that option and found nothing.

**Comment by ge0rdi** [comment, marked as answer]
> I'm wondering if I can accomplish the same effect using only registry entries.

What do you mean by "the same effect"?
Like to have `Open-Shell`-like start menu with just some registry entries?

**No, that's not possible.**

**Comment by DJGray** [comment]
Sorry for not being clear, ge0rdi.

Since I'm not allowed to install Open Shell, I'm wondering if there is a way to get the Windows 7 menu look and feel by manipulating the registry. I suspect not, but wanted to put the question out there.

**Reply by Imold** [reply]
Copy "C:\Program Files\Open-Shell" to a flash drive, it can be run as a mostly portable app.*

*The settings won't carry over with it it, and it will save them in %AppData%.
You could open a Feature Request for a portable installation option.

**Comment by DJGray** [comment]
This is a fascinating and creative proposal. I'd love to try it.

Unfortunately, we're blocked from using flash drives as well.

But I love this creative thinking.
